front/src/metadata/states/apolloClientMetadataState.ts

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import { atom } from 'recoil';
+
+export const apolloClientMetadataState =
+  atom<ApolloClient<NormalizedCacheObject> | null>({
+    key: 'apolloClientMetadataState',
```

Comment:
is this storing the cache? I'm unsure why we need it?

Reply:
This is useless now, it was part of a test